crates/testing/tests/tests_1/test_with_failures_2.rs

```diff
@@ -101,3 +102,48 @@ cross_tests!(
         metadata
     },
 );
+
+#[cfg(not(feature = "dependency-tasks"))]
+cross_tests!(
+    TestName: test_with_double_leader_failures,
+    Impls: [MemoryImpl, Libp2pImpl, PushCdnImpl],
+    Types: [TestConsecutiveLeaderTypes],
+    Ignore: false,
+    Metadata: {
+        let mut metadata = TestDescription::default_more_nodes();
+        metadata.num_bootstrap_nodes = 10;
+        metadata.num_nodes_with_stake = 12;
+        metadata.da_staked_committee_size = 12;
+        metadata.start_nodes = 12;
+        let dead_nodes = vec![
+            ChangeNode {
+                idx: 3,
+                updown: UpDown::Down,
+            },
+        ];
+
+        // shutdown while node 3 is leader
+        // we want to trigger `ViewSyncTrigger`
+        // then ensure we do not fail again as next leader will be leader 2 views also
+        let view_spin_node_down = 5;
+        metadata.spinning_properties = SpinningTaskDescription {
+            node_changes: vec![(view_spin_node_down, dead_nodes)]
+        };
+
+        // node 3 is leader twice when we shut down
+        metadata.overall_safety_properties.num_failed_views = 2;
+        metadata.overall_safety_properties.expected_views_to_fail = HashMap::from([
+            // next views after turning node off
+            (ViewNumber::new(view_spin_node_down+1), false),
```

Comment:
Looks like the formatter didn't run here. Hit this with `rustfmt crates/**/*.rs` and it'll fix this file up.